### ISSUE TITLE
Twilio in Expo App

### ISSUE THREAD

**Issue by prj786** [blackuy/react-native-twilio-video-webrtc#726]
### Steps to reproduce
1. i installed this package
2. followed steps
3. start app

### Expected behaviour
expected app to be started

### Actual behaviour
i am getting an error: RNCCustomTwilioVideoView was not found in the UIManager

### Environment
- **Node.js version**: 18.15.0
- **React Native version**:  0.71.8
- **React Native platform + platform version**: iOS 9.0

### react-native-twilio-video-webrtc
**Version**:  "3.2.0"

So basically I am not react-native developer, But at the moment i need to implement twilio-video in my app. We are using expo. can you show me exact steps to follow? It would be perfect if it works fine in Expo Go. I did a lot of things nothing worked as expected.

Thanks in advance

**Comment by Umar3310** [comment]
Can't use this library in Expo's Managed workflow
Step 1:
Install expo-build-properties
`npx expo install expo-build-properties`

Step 2:
Add the following in plugins in app.json
`[
      "react-native-twilio-video-webrtc",
      {
        "cameraPermission": "Allow $(PRODUCT_NAME) to access your camera",
        "microphonePermission": "Allow $(PRODUCT_NAME) to access your microphone"
      }
    ]`
    
    and 
    
    `[
        "expo-build-properties",
        {
          "android": {
            "compileSdkVersion": 33,
            "targetSdkVersion": 33,
            "buildToolsVersion": "31.0.0"
          },
          "ios": {
            "deploymentTarget": "13.0"
          }
        }
      ]`
    
Setp 3:
Prebuild your project
Follow: [Prebuild Docs](https://docs.expo.dev/workflow/prebuild/)

Step 4:
Configure the project according to the library's instructions 

- [Andriod](https://github.com/blackuy/react-native-twilio-video-webrtc#android)
- [IOS](https://github.com/blackuy/react-native-twilio-video-webrtc#android)

Step 5:
Create a development build
Follow: [Docs](https://docs.expo.dev/develop/development-builds/installation/)

**Comment by flexbox** [comment]
Hello @prj786 

If your problem is solved can you please close the issue? 
It sparks joy for maintainers ✨
